docs/src/content/docs/ko/guides/i18n.mdx

```diff
@@ -391,12 +391,13 @@ const arabicDirection = Astro.locals.t.dir('ar');
 
 [`Astro.currentLocale`](https://docs.astro.build/ko/reference/api-reference/#astrocurrentlocale)을 사용하여 `.astro` 컴포넌트의 현재 로케일을 읽을 수 있습니다.
 
-다음 예시는 현재 로케일을 읽고 이를 사용하여 현재 언어로 작성된 소개 페이지로 이동하는 링크를 생성합니다.
+다음 예시는 현재 로케일을 읽고 이를 [`getRelativeLocaleUrl()`](https://docs.astro.build/en/reference/api-reference/#getrelativelocaleurl) 헬퍼와 함께 사용하여 현재 언어로 된 소개 페이지 링크를 생성합니다:
```

Comment:
```suggestion
다음 예시는 현재 로케일을 읽고 이를 [`getRelativeLocaleUrl()`](https://docs.astro.build/ko/reference/modules/astro-i18n/#getrelativelocaleurl) 헬퍼와 함께 사용하여 현재 언어로 된 소개 페이지 링크를 생성합니다:
```

I'm not a native speaker, but don't you wanna link to the same locale?

Also, I was so cheeky and updated the link to the new Astro Page for you (#2524). Good job for finding this @liruifengv btw...